samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -66,6 +66,31 @@ void client_acknowledge(void)
 	lwm2m_acknowledge(&client);
 }
 
+static void security_handler(enum security_event_type event)
+{
```

Comment:
Can you move this default handler into the `lwm2m_security.c` and then allow the `lwm2m_init_security()` to be called with handler being NULL.